### DISCUSSION TITLE
Error attempting to install current version of pyodbc on Debian Buster (32-bit) with Python 3.7

### DISCUSSION THREAD

**Discussion by SymbReprUnlim** [mkleehammer/pyodbc#1176] [Q&A] [answered]
- Python: 3.7
- pyodbc: 4.0.35
- OS: Debian Buster
- DB: hopefully access.mdb files
- driver:

Error attempting to install current version of pyodbc using  $ pip install pyodbc  on Debian Buster using Python 3.7.
Also, $ pip3 install pyodbc   similarly fails. The only successful install of pyodbc
on Debian seems to be with the sudo apt-get install command as per:
$ sudo apt-get install python3-pyodbc python3-pyodbc-dbg
However, the above sudo command only installs a 4.5 year old or so pyodbc version 4.0.22:
Setting up python3-pyodbc (4.0.22-1+b1) ...
Setting up python3.7-dbg (3.7.3-2+deb10u4) ...

Any helpful suggestions would be appreciated! Thank you.
Error details below:
----------------------------------------------------------------------------------------------------------
$ pip install pyodbc
Defaulting to user installation because normal site-packages is not writeable
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting pyodbc
  Using cached pyodbc-4.0.35.tar.gz (273 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: pyodbc
  Building wheel for pyodbc (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  Building wheel for pyodbc (pyproject.toml) did not run successfully.
  exit code: 1
  
  [14 lines of output]
  running bdist_wheel
  running build
  running build_ext
  building 'pyodbc' extension
  creating build
  creating build/temp.linux-i686-cpython-37
  creating build/temp.linux-i686-cpython-37/src
  i686-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -flto -fuse-linker-plugin -ffat-lto-objects -fPIC -DPYODBC_VERSION=4.0.35 -I/usr/include/python3.7m -c src/buffer.cpp -o build/temp.linux-i686-cpython-37/src/buffer.o -Wno-write-strings
  In file included from src/buffer.cpp:12:
  src/pyodbc.h:56:10: fatal error: sql.h: No such file or directory
   #include <sql.h>
            ^~~~~~~
  compilation terminated.
  error: command '/usr/bin/i686-linux-gnu-gcc' failed with exit code 1
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyodbc
Failed to build pyodbc
ERROR: Could not build wheels for pyodbc, which is required to install pyproject.toml-based projects

**Comment by gordthompson** [comment, marked as answer]
The fix for a missing "sql.h" file is to `apt install unixodbc-dev`
